tsdb/querier.go

```diff
@@ -64,7 +64,7 @@ func (q *querier) LabelNames() ([]string, storage.Warnings, error) {
 
 func (q *querier) lvals(qs []storage.Querier, n string) ([]string, storage.Warnings, error) {
 	if len(qs) == 0 {
-		return nil, nil, nil
+		return []string{}, nil, nil
```

Comment:
I'm not sure if the changes in this method `lvals` break backwards compatibility.  

Before this change if this code block was executed, the api would return `[]string(nil)`, but that was inconsistent with what most nil cases would return as `[]string{}`.  Is this an ok change to make? Or should I keep it as `[]string(nil)` for compatibility?